### ISSUE TITLE
Add feature: import/export to CSV (PWong)

### ISSUE THREAD

**Issue by wanoz** [pwongy/Timestream#19]
High priority

**Comment by riverol** [comment]
This will probably aid debugging and testing, but is this something you want to keep as a user feature?

**Comment by pwongy** [comment]
@riverol Yes I wanted this mainly for testing, but would like to keep it as a user feature so that users have control over their data.
